tests/js/client/chaos/test-chaos-load-common.inc

```diff
@@ -121,32 +121,33 @@ const checkViewConsistency = (cn, isSearchAlias, isMaterialize) => {
   };
   let expected = runQuery("FOR d IN " + cn + " COLLECT WITH COUNT INTO length RETURN length");
   // TODO uncomment assert when will be fixed SEARCH-419
-  // print("Collection: " + expected + " search: " + isSearchAlias + " materialize: " + isMaterialize);
+  print("Collection: " + expected + " search: " + isSearchAlias + " materialize: " + isMaterialize);
```

Comment:
should we actually print something out here?

Reply:
Why not?

Reply:
Because tests shouldn't print unnecessary output?